web-ui/src/main/resources/catalog/components/elasticsearch/EsService.js

```diff
@@ -92,13 +95,25 @@
           if (luceneQueryString) {
             queryStringParams.push(luceneQueryString);
           }
-          queryHook.must.push({
-            query_string: {
-              query: queryStringParams.join(' AND ').trim()
-            }
-          });
-        }
 
+          if (titleOnly) {
+            var query = gnGlobalSettings.gnCfg.mods.search.queryTitle.replace(
+              /\$\{any\}/g, escapeSpecialCharacters(p.any));
+            queryHook.must.push({
+              query_string: {
+                fields: ["resourceTitleObject.*"],
+                query: exactMatch ? query.replaceAll("*", "") : query // remove wild cards for exact match
+              }
+            });
+          } else {
```

Comment:
This should use the same logic as above in cast of `exactMatch`, e.g. :

```suggestion
          if (titleOnly) {
            var searchString = escapeSpecialCharacters(p.any);
            var query = gnGlobalSettings.gnCfg.mods.search.queryTitle.replace(
              /\$\{any\}/g,
              exactMatch === true ? '\"' + searchString + '\"' : searchString);
            queryHook.must.push({
              query_string: {
                fields: ["resourceTitleObject.*"],
                query: query
              }
            });
          } else {
```

Note that this is untested code!